catalog/transformer/catalog-transformer-pptx/src/test/java/ddf/catalog/transformer/input/pptx/PptxInputTransformerTest.java

```diff
@@ -114,10 +114,10 @@ public void testWithTitle()
                 ss.write(os);
 
                 try (ByteArrayInputStream inStr = new ByteArrayInputStream(os.toByteArray())) {
```

Comment:
Should a test be added for not being able to generate the thumbnail?

Reply:
I looked into it and there are two obstacles. 
1. Though about getting a PowerPoint and dumping it into the `test/resources` folder. Then I could use the file as part of a unit test. The problem is, there is only 1 file in the world that generates this error and it classified `SECRET`.  The files is accessible to the DTL. 
2. Use mock objects. The problem with that, is that the public method on the transformer accepts a an input stream of bytes that represents a PowerPoint file. The methods that extract the thumbnail image are all private. So I'm back to the same issue as #1-- I need a byte stream that decodes to a valid PowerPoint file, but causes an exception when we try to extract a thumbnail.

I'm working with Joe Bancroft from the DTL to test the changes on the 1 bad PowerPoint file we have.